fe/fe-core/src/main/java/org/apache/doris/analysis/InsertStmt.java

```diff
@@ -322,8 +331,15 @@ public void analyze(Analyzer analyzer) throws UserException {
     private void analyzeTargetTable(Analyzer analyzer) throws AnalysisException {
         // Get table
         if (targetTable == null) {
-            DatabaseIf db = Env.getCurrentInternalCatalog().getDbOrAnalysisException(tblName.getDb());
-            targetTable = (Table) db.getTableOrAnalysisException(tblName.getTbl());
+            DatabaseIf db = analyzer.getEnv().getCurrentCatalog().getDbOrAnalysisException(tblName.getDb());
```

Comment:
what is difference between 'getCurrentCatalog' and 'getCurrentInternalCatalog' ?

Reply:
`getCurrentInternalCatalog` only return `InternalCatalog`, no matter what is your current catalog is. But `getCurrentCatalog` will return the real current catalog(set by `switch` cmd)